docs/pages/build/eas-json.mdx

```diff
@@ -23,19 +23,19 @@ description: Learn how a project using EAS services is configured with eas.json.
 
 ## Build profiles
 
-A build profile is a named grouping of configurations that describes the necessary parameters to perform a certain type of build.
+A build profile is a named group of configurations that describes the necessary parameters to perform a certain type of build.
 
-The JSON object under the `build` key can contain multiple build profiles, and you can name these build profiles whatever you like; in the above example, there are three build profiles: `development`, `preview`, and `production`, but these could have been named `foo`, `bar`, and `baz` if that was your preference.
+The JSON object under the `build` key can contain multiple build profiles, and you can name these build profiles whatever you like. In the previous example, there are three build profiles: `development`, `preview`, and `production`. However, these could have been named `foo`, `bar`, and `baz` if that is your preference.
 
 To run a build with a specific profile, execute `eas build --profile <profile-name>`. If you omit the `--profile` flag, EAS CLI will default to using the profile with the name **production**, if it exists.
 
 ### Platform-specific and common options
 
-Inside each build profile you can specify `android` and `ios` fields that contain platform-specific configuration for the build. Fields that are available to both platforms can be provided on the platform-specific configuration object or on the root of the profile.
+Inside each build profile, you can specify the `android` and `ios` fields that contain platform-specific configuration for the build. Fields that are available to both platforms can be provided on the platform-specific configuration object or on the root of the profile.
```

Comment:
```suggestion
Inside each build profile, you can specify `android` and `ios` fields that contain platform-specific configuration for the build. Fields that are available to both platforms can be provided on the platform-specific configuration object or on the root of the profile.
```
I'd prefer it without 'the', as this now implies there are also android and ios fields that contain non-platform-specific configuration.